install.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+# Based on Deno installer: Copyright 2019 the Deno authors. All rights reserved. MIT license.
+
+set -e
+
+case $(uname -sm) in
+"Darwin x86_64") target="Darwin_x86_64" ;;
+"Darwin arm64") target="Darwin_arm64" ;;
+"Linux aarch64") target="Linux_arm64" ;;
+*) target="Linux_i386" ;;
```

Comment:
I tested this on Linux via Docker. I think this is all the cases we need to cover.

I haven't attempted Windows support. I think we should create a separate script if we need that. Maybe with powershell we could also try chocately to have an alternative to homebrew.